### ISSUE TITLE
Could you tell me which version of implementation for calculating the expression error?

### ISSUE THREAD

**Issue by graydove** [zyainfal/One-Shot-Face-Swapping-on-Megapixels#45]
There are two implementations (an official tensorflow version and a pytorch version) of the following paper:

Y. Deng, J. Yang, S. Xu, D. Chen, Y. Jia, and X. Tong, Accurate 3D Face Reconstruction with Weakly-Supervised Learning: From Single Image to Image Set, IEEE Computer Vision and Pattern Recognition Workshop (CVPRW) on Analysis and Modeling of Faces and Gestures (AMFG), 2019. 

Because the accuracy of these implementations is different.
Could you tell me which version of implementation is used in the paper?
 
Thank you

**Comment by lizian1611** [comment]
> 以下论文有两种实现（官方的tensorflow版本和pytorch版本）：
> 
> Y. Deng、J. Yang、S. Xu、D. Chen、Y. Jia 和 X. Tong，弱监督学习下的精确 3D 人脸重建：从单幅图像到图像集，IEEE 计算机视觉和模式识别研讨会（ CVPRW) 关于面部和手势的分析和建模 (AMFG)，2019。
> 
> 因为这些实现的准确性是不同的。 你能告诉我论文中使用了哪个版本的实现吗？
> 
> 谢谢

Have you solved this problem? Can you share your code for calculating post and exp？
